### ISSUE TITLE
Enlarged tectum

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7949]
**Preferred term label:**
Enlarged tectum

**Synonyms**
Thick tectum

**Definition (free text, please give PubMed ID)**
Increasaed size of the tectum, which is the region of the midbrain posterior to the cerebral aqueduct of Sylvius. The midbrain is the most superior portion of the brainstem.
PMID:23518331

**Parent term (use hpo.jax.org/app)**
Abnormal brainstem morphology HP:0002363

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
618325 
PMID:30471716   1/9

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
